front/public/locales/fr/stdcm-help-section.json

```diff
@@ -151,7 +151,7 @@
               "type": "list"
             }
           ],
-          "title": "Tonnage, longueur, vitesse max."
+          "title": "Tonnage, longueur, vitesse max"
```

Comment:
Hum no `.` needed here I think

Reply:
I removed it to match the english string.